Tools/AP_Bootloader/app_comms.h

```diff
@@ -10,7 +10,10 @@
 
 struct app_bootloader_comms {
     uint32_t magic;
-    uint32_t reserved[4];
+    uint32_t ip;
+    uint32_t netmask;
```

Comment:
Couldn't we make this netmask 8bit to reserve 3 more bytes for later?

Reply:
we could, but then we need functions to go between the two, adding complexity. I don't think it is worth it.

Reply:
you mean like these that we already have?
```
    static uint32_t convert_netmask_bitcount_to_ip(const uint32_t netmask_bitcount);
    static uint8_t convert_netmask_ip_to_bitcount(const uint32_t netmask_ip);

```

Once we lose those 3 bytes, they're gone forever

Reply:
I'm OK with the uint32_t if @bugobliterator is too.